app/models/comment.rb

```diff
@@ -1,2 +1,10 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def count_updater(post_id)
+    post = Post.find_by(id: post_id)
+    post.comments_counter = Comments.where(Post_id:).count
+    post.save
+  end
```

Comment:
- [ ] This method can be refactored to improve performance. See my suggestion here 👇🏾 
 You can also call this method using the `after_save` method as I mentioned in my previous comment.

```ruby
# app/models/comment.rb
...

def count_updater # the most suitable name for this method should be -> update_comments_counter
  post.increment!(:comments_counter)
end
```